app/assets/javascripts/item_selector/item_selector_breadcrumbs.js

```diff
@@ -1,11 +1,14 @@
+//= require 'item_selector/item_selector_incrementor'
+
 var itemSelectorBreadcrumbs = (function() {
 
-  var breadcrumbTemplate = function(item) {
+  var breadcrumbTemplate = function(item, extraMarkup) {
     var barcode = item.data('barcode');
     var callnumber = item.data('callnumber');
     return $([
       '<div id="breadcrumb-' + barcode + '" class="breadcrumb-pill">',
         callnumber,
+        extra_markup,
```

Comment:
Identifier 'extra_markup' is not in camel case.<br>'extra_markup' is not defined.